workshop/exercise-6/README.md

```diff
@@ -51,7 +51,7 @@ A Knative BuildTemplate encapsulates a shareable build process with some limited
 	```
 3. Run `kubectl get pods --watch` to see the pods initializing.
 
-4. Take a look at the `service.yaml` file again. The service.yaml file defines the required Knative components to build the application from source code in the git repository, push the built container image to the private container registry, and then run the application with the provide URL. 
+4. Take a look at the `service.yaml` file again. The service.yaml file defines the required Knative components to build the application from source code in the git repository and push the built container image to the private container registry.
```

Comment:
We should say something about it serving the application IMO.

Reply:
fixed